src/component/VideoFilter/VideoFilter.jsx

```diff
@@ -0,0 +1,20 @@
+import "./videoFilter.css";
+import { useCategory } from "../../context";
+
+const VideoFilter = ({ categor }) => {
```

Comment:
I think categor is a typo

Reply:
actually no .....i had a useState named category thats why named it categor